scripts/test_boundary_removal.py

```diff
@@ -0,0 +1,12 @@
+"""Script that prints the boundaryless versions of regexes in json format"""
+import json
+import re
+
+from pywhat.helper import load_regexes
+
+regexes = load_regexes()
+for regex in regexes:
+    regex["Regex"] = re.sub(r"(?<!\\)\^(?![^\[\]]*(?<!\\)\])", "", regex["Regex"])
+    regex["Regex"] = re.sub(r"(?<!\\)\$(?![^\[\]]*(?<!\\)\])", "", regex["Regex"])
+
+print(json.dumps(regexes, indent=3))
```

Comment:
You can run this as a test in CI using GitHub Actions btw if you wanted to :)

Reply:
I created it for debug purposes only, so no